components/list/list-item-generic-layout.js

```diff
@@ -154,7 +154,7 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 			}
 
 			::slotted([slot="content-action"]) {
-				grid-column: content-start / end;
+				grid-column: content-start / content-end;
```

Comment:
Hopefully the vdiffs capture this. Didn't look like anything broken would relate to this.

This no longer worked with the structural changes I made. It also seemed like we didn't want the content action extending over the actions anyways.

Reply:
Yeah that seems to be the case. It looks like the actions are render on top of the primary action layer, and fills the full height.